### PR TITLE
Re-re-structure

### PR DESCRIPTION
**Pull request by DanKGooGLy**
* Restructure the file directory
* Clean up the SCSS files

### CODE REVIEW

**Comment by mugman174** [comment]
imma inspect dis

**Comment by mugman174** [comment]
That's a lot of code that I don't feel like reviewing right now so I'm just gonna do a quick check and pull

**Comment by mugman174** [comment]
its kinda broken...

**Comment by mugman174** [comment]
broken hehe @DanKGooGLy is it supposed to do this

**Comment by DanKGooGLy** [comment]
ok let me fix that

**Comment by mugman174** [comment]
Where jumbotron

File: 404.html [hunk 1]
```diff
@@ -27,7 +27,7 @@ <h1>404</h1>
         </div>
         <div class="content">
            <table style="display:block;"><tr>
-              <td><img src="assets/404.png"></td>
+              <td><img src="_assets/imgs/404.png"></td>
```
Comment (mugman174) [inline 404.html:30]: `/_assets` as an absolute directory right?

**Review by mugman174 — APPROVED** [review]

**Comment by DanKGooGLy** [comment]
Yeah

**Comment by mugman174** [comment]
Jumbotron fixed but not dyslexia

**Comment by DanKGooGLy** [comment]
Man

**Comment by mugman174** [comment]
make sure to triple-check filepaths

**Comment by mugman174** [comment]
_waits_

**Comment by mugman174** [comment]
The normal css loads but the dyslexic font doesn't

**Comment by mugman174** [comment]
ok its all fixed

**Pull request merged**